packages/react/src/VideoPlayer/components/Captions/Captions.module.css

```diff
@@ -0,0 +1,17 @@
+.Captions {
+  max-width: 80ch;
+  padding-inline: var(--base-size-16);
+  margin: var(--brand-VideoPlayer-closedCaption-margin);
+  display: grid;
+  justify-content: center;
+  pointer-events: none;
+}
+
+.Captions__text {
+  color: var(--base-color-scale-white-0);
+  text-align: center;
+  background: rgba(0, 0, 0, 0.75);
```

Comment:
Could we make this a CSS var / token please?

Reply:
Resolved in c891b32a48dd0f6c452a2af6e50d5d398384923a